src/data/finset.lean

```diff
@@ -844,7 +844,7 @@ theorem filter_filter (s : finset α) :
   (s.filter p).filter q = s.filter (λa, p a ∧ q a) :=
 ext $ assume a, by simp only [mem_filter, and_comm, and.left_comm]
 
-@[simp] lemma filter_true {s : finset α} [h : decidable_pred (λ _, true)] :
+lemma filter_true {s : finset α} [h : decidable_pred (λ _, true)] :
```

Comment:
Can you explain this change?

Reply:
The linter reports that `simp` can prove this lemma using the new `filter_true_of_mem`: `by simp only [finset.filter_true_of_mem, finset.filter_congr_decidable, forall_true_iff]`. Is there some better fix for the linter warning in that case than removing the `@[simp]` from the lemma that `simp` can prove?

Reply:
That is the correct behavior.

Reply:
@jsm28 Thanks! I didn't expect that `simp` could already prove this.